### ISSUE TITLE
Is it possible to load only a portion of a page into colorbox?

### ISSUE THREAD

**Issue by ezmiller** [jackmoore/colorbox#690]
Something like:

```
$('a').colorbox({
    href: function() { return $(this).attr('href') + '#somediv'; }
});
```

This hasn't worked for me, but is there something along these lines that might be possible?

**Comment by aik099** [comment]
Looks like you're trying to full modal window with contents of div with `somediv` ID, right?

Then you'll need to use `inline` content type. You can see an example on docs page: http://www.jacklmoore.com/colorbox/

**Comment by ezmiller** [comment]
@aik099 : that's right. however, as i understand it, the inline option works only when referencing `somediv` on the same page on which colorbox is being called. i'm trying to point colorbox to another page -- the href of the anchor it is called on -- but only display a portion `somediv` of that page.

**Comment by ezmiller** [comment]
here is the related stackoverflow question: https://stackoverflow.com/questions/27984747/is-it-possible-to-load-only-part-of-a-page-using-colorbox

**Comment by aik099** [comment]
I don't like all this inline div dances. Can't you create separate pages with only contents of these divs inside and access them through ajax calls?

**Comment by ezmiller** [comment]
maybe, but this requires much more work. i was trying to find an easy solution. if this were possible, it wouldn't even be a hacky solution. it seems like a pretty logical function for colorbox not only to be able to load a page, but to load a specific section of a page -- although maybe it's a rather unique use-case. incidentally, jquery ajax can do this i think.

**Comment by aik099** [comment]
> maybe, but this requires much more work. 

Really? Moving out div content into separate HTML file and then setting url to it. Of course, when creating static website this might be a big deal, but when doing something where PHP is involved not so much.

> although maybe it's a rather unique use-case. incidentally, jquery ajax can do this i think.

Looks more like it.

**Comment by jackmoore** [comment]
As was mentioned in the Stack Overflow post, Colorbox uses jQuery's load method for it's ajax handling, so the same rules apply (add a space between the end of your url and the selector).  For example:

```
$('a').colorbox({
    href: function() { return $(this).attr('href') + ' #somediv'; }
});
```

I've never tried this when the url was the same as my current page, but I don't see why it wouldn't work.  Typically, you would use Colorbox's inline property, or clone the markup and use Colorbox's html property, rather than making another http request.

If you put the space before the selector and it still doesn't work, then my guess would be that you are not testing on a server.  The page needs to be served from a webserver in order for ajax to work properly, rather than just opening the html file location in your browser.

**Comment by jackmoore** [comment]
That, or possibly the markup is invalid or some other unknown.  It sounds like an issue that could be solved quickly if you could post a link that demos of the problem.
